src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -147,19 +147,14 @@ if [ "$build_arch" == "armel" ]; then
     cmake_extra_defines="$cmake_extra_defines -DARM_SOFTFP=1"
 fi
 
-clang_version=$( $CC --version | head -1 | sed 's/[^0-9]*\([0-9]*\.[0-9]*\).*/\1/' )
-# Use O1 option when the clang version is smaller than 3.9
-# Otherwise use O3 option in release build
-if [[ ( ${clang_version%.*} -eq 3  &&  ${clang_version#*.} -lt 9 ) &&
-      (  "$build_arch" == "arm" || "$build_arch" == "armel" ) ]]; then
-    overridefile=clang-compiler-override-arm.txt
-else
-    overridefile=clang-compiler-override.txt
-fi
-
 __currentScriptDir="$script_dir"
 
-cmake_command=cmake
+if command -v "cmake3" > /dev/null 2>&1
```

Comment:
A nit - this can be simplified to cmake_command=$(command -v cmake3 || command -v cmake)